tools/ide_plugins/pysa-vscode/src/main.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * @file Entry point for Pyre's VSCode extension.
+ */
+
+import * as vscode from 'vscode';
+import { LanguageClient, LanguageClientOptions, DidChangeConfigurationNotification } from 'vscode-languageclient';
+
+let languageClient: LanguageClient;
+
+namespace Configuration {
+
+	let configurationListener: vscode.Disposable;
+
+	export function initialize() {
+		configurationListener = vscode.workspace.onDidChangeConfiguration(() => {
+			languageClient.sendNotification(DidChangeConfigurationNotification.type, { settings: null });
+		});
+	}
+
+	export function dispose() {
+		if (configurationListener) {
+			configurationListener.dispose();
+		}
+	}
+}
+
+export async function activate(_: vscode.ExtensionContext) {
+
+    let debugOptions = { execArgv: ["--nolazy", "--inspect=6009"] };
+
+    let serverOptions = {
+	command: "pyre",
```

Comment:
just adding on @gbleaney: here you are using a tab but the previous line is using spaces. Let's try to be consistent :)

Reply:
I just realised VScode wasn't converting my tabs to 4 spaces. Fixed it.